packages/actor-scraper/cheerio-scraper/package.json

```diff
@@ -0,0 +1,37 @@
+{
+	"name": "actor-cheerio-scraper",
+	"version": "2.0.0",
+	"description": "Crawl web pages using HTTP requests and Cheerio",
+	"main": "main.js",
+	"dependencies": {
+		"@apify/scraper-tools": "^0.4.4",
```

Comment:
if this is now package in monorepo, we should align its version with other packages